argo/common/common_types_unit_test.go

```diff
@@ -0,0 +1,120 @@
+//go:build unit
+
+package common
+
+import (
+	"encoding/json"
+	
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Context("Marshal NamespacedName", Serial, func() {
```

Comment:
These seem unrelated? If we added them here for completeness we should say so in the PR description

Reply:
Sorry, this was from my suggestion to refactor the existing tests that were all in suite_unit_test and to follow the same pattern as everywhere else and have own test files

Reply:
I have added a note to the description. 👍🏼